tests/sanity/ignore-2.15.txt

```diff
@@ -8,3 +8,92 @@ plugins/modules/logs_metric_filter.py validate-modules:mutually_exclusive-type
+plugins/modules/rds_global_cluster.py validate-modules:parameter-state-invalid-choice
+plugins/modules/rds_option_group.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ssm_document.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ssm_resource_data_sync.py validate-modules:no-log-needed
+plugins/modules/ssm_resource_data_sync.py validate-modules:parameter-state-invalid-choice
+plugins/modules/wafv2_ip_set.py validate-modules:parameter-state-invalid-choice
+plugins/modules/wafv2_logging_configuration.py validate-modules:parameter-state-invalid-choice
+plugins/modules/wafv2_regex_pattern_set.py validate-modules:parameter-state-invalid-choice
+plugins/modules/wafv2_web_acl_association.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ecr_repository.py validate-modules:no-log-needed
+plugins/modules/ecr_repository.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ecs_capacity_provider.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ecs_cluster.py validate-modules:no-log-needed
+plugins/modules/ecs_cluster.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ecs_cluster_capacity_provider_associations.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ecs_primary_task_set.py validate-modules:parameter-state-invalid-choice
+plugins/modules/iam_instance_profile.py validate-modules:parameter-state-invalid-choice
+plugins/modules/logs_metric_filter.py validate-modules:no-log-needed
+plugins/modules/logs_metric_filter.py validate-modules:parameter-state-invalid-choice
+plugins/modules/ec2_placement_group.py validate-modules:parameter-state-invalid-choice
+plugins/modules/autoscaling_launch_configuration.py validate-modules:no-log-needed
+plugins/modules/autoscaling_launch_configuration.py validate-modules:parameter-state-invalid-choice
+plugins/modules/autoscaling_lifecycle_hook.py validate-modules:parameter-state-invalid-choice
+plugins/modules/autoscaling_warm_pool.py validate-modules:parameter-state-invalid-choice
+plugins/modules/eks_addon.py validate-modules:mutually_exclusive-type
+plugins/modules/eks_fargate_profile.py validate-modules:mutually_exclusive-type
+plugins/modules/redshift_endpoint_authorization.py validate-modules:mutually_exclusive-type
+plugins/modules/route53_key_signing_key.py validate-modules:mutually_exclusive-type
+plugins/modules/autoscaling_lifecycle_hook.py validate-modules:mutually_exclusive-type
+plugins/modules/ecs_primary_task_set.py validate-modules:mutually_exclusive-type
+plugins/modules/logs_metric_filter.py validate-modules:mutually_exclusive-type
+plugins/modules/wafv2_ip_set.py validate-modules:mutually_exclusive-type
+plugins/modules/wafv2_regex_pattern_set.py validate-modules:mutually_exclusive-type
+plugins/modules/wafv2_web_acl_association.py validate-modules:mutually_exclusive-type
+plugins/module_utils/core.py pylint!skip
```

Comment:
I guess we can remove this one.

Reply:
@alinabuzachis  Added this line to skip `ERROR: plugins/module_utils/core.py:46:0: unused-import: Unused ansible_dict_to_boto3_tag_list imported from utils`.

Reply:
@GomathiselviS Something like this should be enough https://github.com/ansible-collections/amazon.aws/blob/main/plugins/module_utils/ec2.py#L46 It shouldn't be necessary to skip the whole file.